python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1245,7 +1254,15 @@ def get_edge_data(self, edge_ids=None, types=None, columns=None):
                 df = df[
                     [self.src_col_name, self.dst_col_name, self.type_col_name] + columns
                 ]
-            return df.reset_index()
+            df_out = df.reset_index()
```

Comment:
I asume you dont want the index to be lying around in memory
```suggestion
            df_out = df.reset_index(drop=True)
```